assets/component-cart-items.css

```diff
@@ -130,6 +147,7 @@ cart-remove-button .icon-remove {
 .cart-item__error {
   font-size: 1.2rem;
   display: flex;
+  align-items: baseline;
```

Comment:
<img width="417" alt="Screen Shot 2021-08-23 at 11 33 20 AM" src="https://user-images.githubusercontent.com/26778637/130475249-400919f5-7561-4709-90a6-b35b073426f6.png">

Since you are already modifying the error styling a bit, can we add a `flex-shrink: 0` to the error message icon to prevent it from shrinking?

Reply:
Great suggestion. I fixed it